content/docs/user-guide/project-structure/dvcyaml-files.md

```diff
@@ -34,6 +30,21 @@ artifacts:
       framework: pytorch
 ```
 
+For every artifact ID you can specify the following elements (only `path` is
+mandatory):
+
+- `path` (_string_) - The path to the artifact, either relative to the root of
+  the repository or a full path in an external storage such as S3.
+- `type` (_string_) - You can specify artifacts of any `type` and the DVC-based
+  [model registry](/doc/use-cases/model-registry) will pick up any artifacts
+  with type `model`. Additionally, they will be available in
+  [Studio Model Registry](/doc/studio/user-guide/model-registry/what-is-a-model-registry)
```

Comment:
This is not coming from the changes in the P.R. but I do not fully understand that is the difference between `DVC-based model registry` and `Studio Model Registry` cc @iterative/docs

Reply:
This was a confusion I think. This is the same registry, no difference.

Reply:
I edited the links, but really I would like to link to a MR tutorial instead of either of these two pages. One is specific to  Studio (I used it now for both links), the other one is just a general page describing what a model registry is with a link to the Studio MR page at the bottom :-)

Reply:
Opened https://github.com/iterative/dvc.org/issues/4836, but I think it's out of scope of this PR.